src/linter/index.mjs

```diff
@@ -58,6 +66,87 @@ const createLinter = (dryRun, disabledRules) => {
     }
   };
 
+  /**
+   * Parse an inline-declaration found in the markdown input
+   *
+   * @param {string} declaration
+   */
+  const parseLinterDeclaration = declaration => {
+    // Trim off any excess spaces from the beginning & end
+    declaration = declaration.trim();
+
+    // Extract the name for the declaration
+    const [name, ...value] = declaration.split(' ');
+
+    switch (name) {
+      case 'skip-deprecation': {
```

Comment:
Wdyt of defining these definitions similarly to the rules & reporters? (i.e. have them defined in a `declarations/skip-deprecation.mjs`, then here we look them up & call them if they exist)